### ISSUE TITLE
Add API docs for v1.1

### ISSUE THREAD

**Issue by vickyhella** [harvester/docs#389]
After #388 is merged, we need to submit another PR to add API reference docs for v1.1.

v1.1 swagger.json location: https://github.com/harvester/harvester/blob/v1.1/api/openapi-spec/swagger.json
Official Documentation: https://docusaurus-openapi.tryingpan.dev/#versioning-openapi-docs

**Comment by vickyhella** [comment]
Closing as this issue was fixed in #388
